spec/features/idv/in_person_spec.rb

```diff
@@ -412,6 +417,30 @@
     end
   end
 
+  context 'validate_id_and_residential_addresses feature flag enabled', allow_browser_log: true do
+    let(:user) { user_with_2fa }
+
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_capture_secondary_id_enabled).
+        and_return(true)
+    end
+
+    it 'captures the address, address line 2, city, state and zip code' do
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      search_for_post_office
+
+      # location page
+      location = page.find_all('.location-collection-item')[1]
+      location.click_button(t('in_person_proofing.body.location.location_button'))
+
+      # prepare page
+      complete_prepare_step(user)
+
+      complete_state_id_step_when_id_and_residential_address_differ
```

Comment:
i think it would be good after this complete step to include an expect that the next page has path :address

Reply:
Sure, I'll add that expectation to `complete_state_id_step_when_id_and_residential_address_differ`.